lib/core/enums/video_auto_play.dart

```diff
@@ -0,0 +1,11 @@
+enum VideoAutoPlay {
+  never(label: 'Never'),
```

Comment:
Can you see about loading these strings from localization? You might have to add a property/method which does a dynamic lookup of the resources using the global context.